.rubocop.yml

```diff
@@ -61,6 +61,11 @@ RSpec/MultipleExpectations:
 Style/Documentation:
   Enabled: false
 
+Style/EndOfLine:
```

Comment:
This was for that weird thing where it was complaining about an extra carriage return, right? What a dumb error...